src/client/theme/SearchBar/SuggestionTemplate.ts

```diff
@@ -42,7 +44,19 @@ export function SuggestionTemplate({
       tokens
     )}</span>`,
   ];
-  if (!isTitle) {
+
+  const needsExplicitHitPath =
+    !isInterOfTree && !isLastOfTree && explicitSearchResultPath;
+  if (needsExplicitHitPath) {
+    const pathItems = page
+      ? page.b?.concat(page.t).concat(document.s ?? [])
```

Comment:
Is it better to make `page.t` and `document.t` still work when `page.b` doesn't exist?

```suggestion
      ? (page.b ?? []).concat(page.t).concat(document.s ?? [])
```

Reply:
Yes, totally, thank you!